akka-actor/src/main/scala/akka/actor/package.scala

```diff
@@ -7,6 +7,6 @@ package akka
 import language.implicitConversions
 
 package object actor {
-  implicit def actorRef2Scala(ref: ActorRef): ScalaActorRef = ref.asInstanceOf[ScalaActorRef]
-  implicit def scala2ActorRef(ref: ScalaActorRef): ActorRef = ref.asInstanceOf[ActorRef]
+  @inline implicit final def actorRef2Scala(ref: ActorRef): ScalaActorRef = ref.asInstanceOf[ScalaActorRef]
+  @inline implicit final def scala2ActorRef(ref: ScalaActorRef): ActorRef = ref.asInstanceOf[ActorRef]
```

Comment:
doesn't hurt but will not make any difference since we don't compile with `-optimise` flag